README.md

```diff
@@ -31,14 +31,14 @@ The following plugins are enabled on the demo platform:
 - tutor-android ([PR](https://github.com/overhangio/tutor-android/pull/5) by @regisb): app can be downloaded from https://mobile.olive.demo.overhang.io/app.apk.
 - tutor-forum ([PR](https://github.com/overhangio/tutor-forum/pull/11) by @ghassanmas)
 - tutor-notes ([PR](https://github.com/overhangio/tutor-notes/pull/18) by @jfavellar90)
+- tutor-xqueue ([PR](https://github.com/overhangio/tutor-xqueue/pull/13) by @jfavellar90)
```

Comment:
I'll merge this as it is to install the plugin, but could you please add extra information to the README, such as the password?